assembly/fabric8-ide-dashboard-war/src/assets/branding/product.json

```diff
@@ -0,0 +1,39 @@
+{
+  "title": "Eclipse Che ",
+  "name": "Eclipse Che",
+  "logoFile": "che-logo.svg",
+  "logoTextFile": "che-logo-text.svg",
+  "favicon": "favicon.ico",
+  "loader": "loader.svg",
+  "websocketContext": "/api/websocket",
+  "helpPath": "https://www.eclipse.org/che/",
+  "helpTitle": "Community",
+  "supportEmail": "che-dev@eclipse.org",
+  "oauthDocs": "Configure OAuth in the che.properties file.",
+  "workspace": {
+    "priorityStacks": ["Java", "Java-MySQL", "Blank"],
+    "defaultStack": "java-mysql"
```

Comment:
The quick fix (just adds there): 
```
"creationLink": "#/create-workspace"
```

Reply:
It is a default value.  But, it won't be used until merge this [PR](https://github.com/eclipse/che/pull/16175)

Reply:
@olexii4 thanks for the pointer!
@vparfonov could you please apply the suggestion to unblock the PR?

Reply:
done